src/lsstseries/ensemble.py

```diff
@@ -391,6 +408,69 @@ def batch(self, func, *args, meta=None, use_map=True, compute=True, on=None, **k
         else:
             return batch
 
+    def from_hipscat(
```

Comment:
I'm not sure there is anything here that is actually specific to hipscat (except possible the separate source and object directories). We could call this something like "from_parquet_directories" and make it more general.